src/po/Make_mvc.mak

```diff
@@ -6,26 +6,55 @@
 # Please read README_mvc.txt before using this file.
 #
 
-!ifndef VIMRUNTIME
-VIMRUNTIME = ..\..\runtime
-!endif
+!IF [powershell -nologo -noprofile "exit $$psversiontable.psversion.major"] == 2
+!ERROR The program "PowerShell" version 3.0 or higher is required to work
+!ENDIF
+
+!IFNDEF LANGUAGE
+!ERROR Set the environment variable %LANGUAGE%. See README_mvc.txt
+!ENDIF
```

Comment:
I'm rethinking whether this check is needed or not.
This check breaks my build script.

`$(LANGUAGE)` is used only in three make targets: `first_time`, `install`, `cleanup-po`.
Other make targets don't need this at all.

Reply:
See #13567

Reply:
vim-win32-installer is also broken: https://ci.appveyor.com/project/chrisbra/vim-win32-installer/build/job/a5rw0d0y1a4e9498